### DISCUSSION TITLE
Newest experimental builds don't have a windows version.

### DISCUSSION THREAD

**Discussion by Aneastin** [community/community#4240] [Codespaces] [answered]
Just noticed when I went to get the latest experimental release and didn't find the windows version of the build, I also couldn't find any post about it either. Noticed that a earlier build does have the windows version so I will be using that for the time being but I was wondering when or if the newest experimental versions will be updated with a windows version. Note the attachment below, the builds listed with 9 assets don't have their windows version.
[Releases · CleverRaven_Cataclysm-DDA · GitHub.pdf](https://github.com/github/feedback/files/6699739/Releases.CleverRaven_Cataclysm-DDA.GitHub.pdf)

**Comment by 2percentsilk** [comment, marked as answer]
Hi @Aneastin,

While Codespaces had a private beta for Windows support earlier this year, that beta has since ended and is no longer available for use.

**Reply by Aneastin** [reply]
So will there be no support for windows going forward? I was trying to get an updated experimental version as dropping anything crashes the game.

**Reply by 2percentsilk** [reply]
Hi @Aneastin,
Rereading your issues, I'm a little confused - are you talking about the builds and releases for the [Cataclysm-DDA](https://github.com/CleverRaven/Cataclysm-DDA/releases) repo? If so, please report problems with the build on that repo.

This forum is meant for [GitHub Codespaces](https://github.com/features/codespaces) feedback. If you're trying to open the [Cataclysm-DDA](https://github.com/CleverRaven/Cataclysm-DDA/releases) repo in a codespace and it requires a Windows machine, that is not supported and not currently on our roadmap.
